emoji-ui/src/components/emojiItem.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import './emojiItem.css';
+import './EmojiItem.css';
```

Comment:
is this change correct? git isn't recognizing the file name as having been changed.